src/main/java/com/joejoe2/chat/exception/UserDoesNotExist.java

```diff
@@ -1,7 +1,11 @@
 package com.joejoe2.chat.exception;
 
 public class UserDoesNotExist extends Exception {
```

Comment:
Same about extending a RuntimeException and being called `UserDoesNotExistException`